File: deno/src/base/options/options.service.ts
import * as optionsSrcDao from "./options.dao.ts";

/**
 * 获取系统选项
 * @param lbl 
 */
export async function getOptionsByLbl(
  lbl: string,
) {
  return await optionsSrcDao.getOptionsByLbl(lbl);
}

File: rust/src/src/base/options/options_service.rs
use color_eyre::eyre::Result;

use super::options_dao;
use crate::r#gen::base::options::options_model::OptionsModel;

pub async fn get_options_by_lbl(
  lbl: String,
) -> Result<Vec<OptionsModel>> {
  
  let res = options_dao::get_options_by_lbl(
    lbl,
  ).await?;
  
  Ok(res)
}
